### PR TITLE
906 start end date validation

### LINKED ISSUE #906: [TASK] | Catalog Talent start date and end date needs validation
### Description
The start date and end date in Talents does not have any validation. Add data validation by linking the start date and end dates field and check if end date is after start date.

### Acceptance Criteria
Start date and end date valid in Talents.

### PR DESCRIPTION
**Pull request by minump**
## Description
Talent start and end date validation is done with this PR.
An onchange() function is implemented in #start_day which sets the min value of #end_day to #start_day.value with setAttribute() method.
Two javascript functions are added - one at the bottom with all js functions and one inline with html edit page.
**Fixes #906**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
@minump, please fix the merge conflicts when you get a chance. The CHANGELOG entry for this PR needs to go at the end of the Added section. Thanks.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Version endpoint to contribution building block  [#897](https://github.com/rokwire/rokwire-building-blocks-api/issues/897)
 - Version endpoint to logging building block  [#898](https://github.com/rokwire/rokwire-building-blocks-api/issues/898)
 - Endpoint for showing required building blocks to contribution building block [#904](https://github.com/rokwire/rokwire-building-blocks-api/issues/904)
+- Start and end date validation for Talents [#906](https://github.com/rokwire/rokwire-building-blocks-api/issues/906)
```
Comment (sandeep-ps) [inline CHANGELOG.md:15]: Please remove this duplicate entry. Thanks.

**Comment by sandeep-ps** [comment]
Hi, @minump, 

Add Talent/Capability button is submitting the Contribution instead of displaying relevant form entries. I'm not sure if something got messed up during the merge process. Please look into this when you get a chance. Thanks.

**Comment by minump** [comment]
Converting to draft as [PR#924](https://github.com/rokwire/rokwire-building-blocks-api/pull/924) is created for the same.

**Comment by sandeep-ps** [comment]
Closing this #924 is merged. Thanks.

**Pull request closed without merging**